frontend/src/components/login.js

```diff
@@ -1,44 +1,165 @@
 import { useState } from 'react';
-import { loginUser } from '../http/user'
+import { registerUser, loginUser } from '../http/user'
 
-const Login = ({ setUserState }) => {
+const Login = ({ setUserState, setMessage, setReturnError }) => {
+    //login states
     const [userInput, setUserInput] = useState('');
     const [passwordInput, setPasswordInput] = useState('');
-    const [loginError, setLoginError] = useState('');
+    //register states
+    const [userRegInput, setUserRegInput] = useState('');
+    const [passwordRegInput, setPasswordRegInput] = useState('');
+    const [emailRegInput, setEmailRegInput] = useState('');
+    //error and toggler states
+    const [errorState, setErrorState] = useState('');
+    const [showRegister, setShowRegister] = useState(false)
 
 
+    //login function
```

Comment:
not sure this comment adds value here. What does "login function" tell a user that "handleLogin = evt =>" doesn't? Comments should be useful, on the other hand, I think the function name could change to 'onLoginButtonClick'.

Reply:
removed

Reply:
changed function name